src/features/movies/MoviePage/index.js

```diff
@@ -1,49 +1,101 @@
-import BigMovieTile from "../../../common/BigMovieTile";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import BigMovieTile from "./BigMovieTile";
 import { Container } from "../../../common/Container";
+import ErrorPage from "../../../common/ErrorPage";
 import { Layout } from "../../../common/Layout/styled";
-import MovieTile from "../../../common/MovieTile";
+import Loader from "../../../common/Loader";
+import MovieTile from "./MovieTile";
 import PageHeader from "../../../common/PageHeader";
 import { PersonTile } from "../../../common/PersonTile";
+import { TileLink } from "../../../common/TileLink";
+import { APIImageUrl } from "../../dataAPI";
+import {
+  fetchMovieId,
+  selectError,
+  selectLoading,
+  selectMovie,
+  selectMovieCredits,
+} from "./movieSlice";
 import { Wrapper } from "./styled";
+import { getPeople } from "../../utilities";
 
 const MoviePage = () => {
+  const dispatch = useDispatch();
+  const movie = useSelector(selectMovie);
+  const movieCredits = useSelector(selectMovieCredits);
+  const error = useSelector(selectError);
+  const loading = useSelector(selectLoading);
+  const params = useParams();
+
+  useEffect(() => {
+    dispatch(fetchMovieId(params.id));
+  }, [dispatch, params.id]);
+
   return (
     <>
-      <BigMovieTile />
-      <Container>
-        <MovieTile />
-        <Wrapper>
-          <PageHeader title="Cast" />
-          <Layout>
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-          </Layout>
-        </Wrapper>
-        <Wrapper>
-          <PageHeader title="Crew" />
-          <Layout>
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-            <PersonTile />
-          </Layout>
-        </Wrapper>
-      </Container>
+      {loading ? (
+        <Loader />
+      ) : error || !movie?.id ? (
+        <ErrorPage />
+      ) : (
+        <>
+          <BigMovieTile
+            poster={
+              movie.backdrop_path &&
+              `${APIImageUrl}/original${movie.backdrop_path}`
+            }
```

Comment:
Just to remind you in this place - movie title, voteAverage (with star icon) and voteCount need to be entered on the backdrop poster.